priv/scripts/hex_docs.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# $1 = event
+# $2 = ref
+
+set -e -u
+
+source ${SCRIPT_DIR}/utils.sh
+
+# $1 = ref
+function build {
+  image="gcr.io/hexpm-prod/bob-hex-docs"
+  container="hex-docs"
+
+  docker pull ${image} || true
+  docker build -t ${image} -f ${SCRIPT_DIR}/hex-docs.dockerfile ${SCRIPT_DIR}
+  docker push ${image}
+  docker rm ${container} || true
+  docker run -t --name=${container} ${image}
+}
+
+# $1 = ref
+function push {
+  app=hex
+  version=$(echo "${1}" | sed -e 's/^v//g' | sed -e 's/\//-/g')
+
+  pushd versioned-docs
+  aws s3 cp "${app}" "s3://hexdocs.pm/${app}/${version}" --recursive --cache-control "public,max-age=3600" --metadata "{\"surrogate-key\":\"docspage/${app}/${version}\",\"surrogate-control\":\"public,max-age=604800\"}"
+  fastly_purge $BOB_FASTLY_SERVICE_HEXDOCS "docspage/${app}/${version}"
+
+  tar -czf "${app}-${version}.tar.gz" -C "${app}" .
+  aws s3 cp "${app}-${version}.tar.gz" "s3://s3.hex.pm/docs/${app}-${version}.tar.gz" --cache-control "public,max-age=3600" --metadata "{\"surrogate-key\":\"docs/${app}-${version}\",\"surrogate-control\":\"public,max-age=604800\"}"
+  fastly_purge $BOB_FASTLY_SERVICE_HEXPM "docs/${app}-${version}"
+  popd
+
+  if [ -f unversioned-docs ]; then
+    pushd unversioned-docs
+    aws s3 cp "${app}" "s3://hexdocs.pm/${app}" --recursive --cache-control "public,max-age=3600" --metadata "{\"surrogate-key\":\"docspage/${app}\",\"surrogate-control\":\"public,max-age=604800\"}"
+    fastly_purge $BOB_FASTLY_SERVICE_HEXDOCS "docspage/${app}"
+  fi
+}
+
+if [ "$1" == "push" ]; then
```

Comment:
I based this on https://github.com/hexpm/bob/blob/32fe561ceb73058c31afc8097f04770a8b69a372/priv/scripts/elixir_guides.sh#L33 but come to think of it, we should only build docs on `vMAJOR.MINOR.PATCH` tags, not on branches (notably not on master); what's a good place to make this check? here?

Reply:
I would do the check in the elixir job. We have a similar check when building OTP.